TrafficCapture/dockerSolution/README.md

```diff
@@ -110,3 +110,12 @@ A problem occurred evaluating project ':trafficCaptureProxyServer'.
          > No locally installed toolchains match (see https://docs.gradle.org/8.0.2/userguide/toolchains.html#sec:auto_detection) and toolchain download repositories have not been configured (see https://docs.gradle.org/8.0.2/userguide/toolchains.html#sub:download_repositories).
 
 ```
+
+## Development
+
+### Updating Pipfile
+
+When updating Pipfiles, run the following command to update all Pipfile.lock in subdirectories
+```bash
+  find . -type f -name Pipfile -execdir bash -c 'PIPENV_IGNORE_VIRTUALENVS=true pipenv lock --python 3.11' \;
```

Comment:
What does this do the user's current environment? Does it install everything in every pipfile? I can't actually tell from the `pipenv lock` documentation.

Reply:
It doesn't have any side effects in the user environment. Lock just resolves and updates
```
(python3_venv) ➜  dockerSolution git:(simplifyPipfile) ✗ find . -type f -name Pipfile -execdir bash -c 'PIPENV_IGNORE_VIRTUALENVS=true pipenv lock --python 3.11' \;
Locking [packages] dependencies...
Building requirements...
Resolving dependencies...
✔ Success!
Locking [dev-packages] dependencies...
Updated Pipfile.lock (fff897626f6e22e3ccfe3cbd4392514d3a41679372c7eb1e5c790783485441c5)!
Locking [packages] dependencies...
Building requirements...
Resolving dependencies...
✔ Success!
Locking [dev-packages] dependencies...
Building requirements...
Resolving dependencies...
✔ Success!
Updated Pipfile.lock (914625b4e6d78413f4259907f8b7debe21f1b33fe35e11743809688e335e6d28)!
Locking [packages] dependencies...
Building requirements...
Resolving dependencies...
✔ Success!
Locking [dev-packages] dependencies...
Updated Pipfile.lock (333a2a22976ccc7f9bd36b9d12956616361b20f2f51c538ddeb8bc706f9c9834)!
Locking [packages] dependencies...
Building requirements...
Resolving dependencies...
✔ Success!
Locking [dev-packages] dependencies...
Building requirements...

```